lib/jsonapi/view.ex

```diff
@@ -63,12 +63,14 @@ defmodule JSONAPI.View do
   defmacro __using__(opts \\ []) do
     {type, opts} = Keyword.pop(opts, :type)
     {namespace, _opts} = Keyword.pop(opts, :namespace, "")
+    {host, _opts} = Keyword.pop(opts, :host, "")
```

Comment:
If you wanted to have it be application wide you could do Application.get_env(:jsonapi, :host)